src/Test/java/Duke/DeadlineTest.java

```diff
@@ -0,0 +1,21 @@
+package Duke;
+import org.junit.jupiter.api.Test;
+import java.time.LocalDateTime;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DeadlineTest {
+
+    @Test
+    public void testGetBy() {
```

Comment:
Do consider looking into junit naming convention for the tests. Test name should express a specific requirement. Here is a [link](https://medium.com/@stefanovskyi/unit-test-naming-conventions-dd9208eadbea) for some of the naming conventions.